docs/getting-started/user-roadmap-zowe-explorer.md

```diff
@@ -1,121 +1,59 @@
-
-* [**Video: Getting started with Zowe Explorer (Part 1)**](https://youtu.be/G_WCsFZIWt4)
-* [**Video: Getting started with Zowe Explorer (Part 2)**](https://youtu.be/X4oSHrI4oN4)
-
-   These videos help you to get started with Zowe Explorer and show the basic data set use cases.
-
-* [**Zowe Explorer Profiles**](../user-guide/ze-profiles.md)
-
-   This page describes how to create and work with Zowe Explorer profiles. Having a profile enables you to use all functions of the extension, activate the Secure Credential Store plug-in to securely store credentials, and more.
-
-## Using Zowe Explorer
-
-> Zowe skill level: Intermediate
+- Review the system requirements and steps for installing the Zowe Explorer in [**Installing Zowe Explorer**](../user-guide/ze-install.md).
 
-* [**Using Zowe Explorer**](../user-guide/ze-usage.md)
+- Learn how to create and work with [**Zowe Explorer profiles**](../user-guide/ze-profiles.md). A profile enables you to connect to services running on a mainframe, integrate with the API Mediation Layer, and more.
 
-   This page includes usage tips and sample use cases for data sets, USS files, JOBs, and TSO commands. Familiarize yourself with the extension and make the best use of available options and features.
+## Getting the most out of Zowe Explorer
 
-## Extending Zowe Explorer
+- Review [**Using Zowe Explorer**](../user-guide/ze-usage.md) to go over how to use Zowe Explorer in a remote environment and managing credentials.
 
-> Zowe skill level: Advanced
+* Learn how to [**extend Zowe Explorer**](https://github.com/zowe/vscode-extension-for-zowe/wiki/Extending-Zowe-Explorer) to introduce new functionalities.
 
-* [**Extend Zowe Explorer**](https://github.com/zowe/zowe-explorer-vscode/wiki/Extending-Zowe-Explorer)
+- Add CICS functionality and the FTP protocol to the Zowe Explorer VS Code extension with the [**Zowe Explorer CICS Extension**](../user-guide/ze-using-zowe-explorer-cics-ext.md) and the [**Zowe Explorer FTP Extension**](../user-guide/ze-ftp-install-ze-ftp-ext.md).
 
-   Learn how to create extensions for Zowe Explorer to introduce new functionalities.
+- Check out the [**Zowe Explorer GitHub repository**](https://github.com/zowe/vscode-extension-for-zowe#readme) to view the source code for Zowe Explorer and other Zowe Explorer-related extensions.
 
-- [**Zowe Explorer CICS Extension**](../user-guide/ze-using-zowe-explorer-cics-ext.md)
+- Watch the following videos to learn how to get started with Zowe Explorer, and work with data sets: 
 
-   Learn how to install the CICS extension. The extension adds CICS functionality to the Visual Studio Code extension, which lets you interact with CICS regions and programs.
+<iframe class="embed-responsive-item" id="youtubeplayer" title="Getting Started with Zowe" type="text/html" width="100%" height="365" src="https://www.youtube.com/embed/G_WCsFZIWt4" frameborder="0" webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen="true"> </iframe>
```

Comment:
I think we do this just so that we can fit more videos per page, so, no, I don't think it's absolutely necessary, just something we do for a better appearance